ang/civicase/case/details/directives/case-details-header.directive.js

```diff
@@ -5,7 +5,63 @@
     return {
       replace: true,
       restrict: 'E',
-      templateUrl: '~/civicase/case/details/directives/case-details-header.directive.html'
+      templateUrl: '~/civicase/case/details/directives/case-details-header.directive.html',
+      controller: 'civicaseCaseDetailsHeaderController'
     };
   });
+
+  module.controller('civicaseCaseDetailsHeaderController', civicaseCaseDetailsHeaderController);
+
+  /**
+   *
+   * @param {*} $scope scope object of the controller
+   * @param {*} CaseActions case action service
+   * @param {*} WebformsCaseAction webform case action service
+   * @param {*} GoToWebformCaseAction go to webform case action service
+   * @param {*} webformsList configuration for webforms list
+   */
+  function civicaseCaseDetailsHeaderController ($scope, CaseActions,
+    WebformsCaseAction, GoToWebformCaseAction, webformsList) {
+    $scope.webformsAction = CaseActions.findByActionName('Webforms');
+    $scope.isGoToWebformAllowed = isGoToWebformAllowed;
+    $scope.openWebform = openWebform;
+    $scope.getWebformDropdownButtonLabel = getWebformDropdownButtonLabel;
+
+    (function init () {
+      $scope.$watch('item', itemWatcher);
+    })();
+
+    /**
+     * @returns {string} label for webform dropdown button
+     */
+    function getWebformDropdownButtonLabel () {
+      return webformsList.buttonLabel;
+    }
```

Comment:
why not pass this as a scope's property instead of using a function to access it?